README.md

```diff
@@ -68,6 +68,19 @@ Kanban-style boards to manage work items. Integrates well with teams and other s
 | Platform  | macOS, Windows, iOS, Android, web              |
 | URL       | https://trello.com/                            |
 
+### Taiga
+
+Taiga is a project management tool for multi-functional agile teams. It has a rich feature set and at the same time it is very simple to start with through its intuitive user interface.
+
+![Taiga](media/tool-taiga.gif)
+
+| Property  | Value                              |
+|-----------|------------------------------------|
+| Developer | [Kaleidos](https://kaleidos.net/)  |
+| Cost      | Open Source - Freemium SaaS        |
```

Comment:
Let's include a link to pricing here (e.g., Freemium SaaS takes you to the pricing page)

Reply:
Since there is only a homepage, I've added the anchor to the pricing section